docs/usage/using-wp-mock.md

```diff
@@ -55,6 +54,17 @@ Calling `WP_Mock::userFunction()` will dynamically define the function for you i
 return \WP_Mock\Functions\Handler::handleFunction(__FUNCTION__, func_get_args());
 ```
 
+
+{% hint style="warning" %}
+
+**Important!**
+
+Note how in the above test class `WP_Mock\Tools\TestCase` was extended instead of `PHPUnit\Framework\TestCase`. This is because WP_Mock provides a custom test case class that extends PHPUnit's test case class to run properly, as well as while adding additional utility and test methods. You should always extend `WP_Mock\Tools\TestCase` instead of `PHPUnit\Framework\TestCase` when using WP_Mock.
+
```

Comment:
Now that we have more complete documentation in the configuration section, I wonder if we should simplify this and just direct curious users to the other doc for full details.

```suggestion
You should typically extend `WP_Mock\Tools\TestCase` instead of `PHPUnit\Framework\TestCase` when using WP_Mock.
```

Reply:
sounds good